wrapanapi/systems/google.py

```diff
@@ -101,16 +101,24 @@ def _get_state(self):
         return self._api_state_to_vmstate(self.raw['status'])
 
     @property
-    def ip(self):
+    def ip_internal(self):
         self.refresh()
         return self.raw.get('networkInterfaces')[0].get('networkIP')
```

Comment:
can it ever throw index error?

Reply:
Making change here and in `ip`. Simple try/except here, in `ip` I'm going to add defaults to the `get()` call chain so that IndexError is the only exception that will happen on the last index.

Reply:
Have you updated this yet? I don't see it.